cfme/containers/container.py

```diff
@@ -21,7 +22,7 @@
 )
 
 
-class Container(Taggable, SummaryMixin):
+class Container(Taggable):
```

Comment:
I see I did some mess there..
Can we revert all this changes and I'll try to add them again the way I should've?